Makefile

```diff
@@ -50,17 +50,17 @@ help:
 	@echo " * 'install-containerd'	- Install customized containerd to system location"
 	@echo " * 'release'          	- Build release tarball"
 	@echo " * 'push'             	- Push release tarball to GCS"
-	@echo " * 'test'             	- Test cri-containerd with unit test"
-	@echo " * 'test-integration' 	- Test cri-containerd with integration test"
-	@echo " * 'test-cri'         	- Test cri-containerd with cri validation test"
-	@echo " * 'test-e2e-node'    	- Test cri-containerd with Kubernetes node e2e test"
+	@echo " * 'test'             	- Test cri with unit test"
```

Comment:
s/cri/cri plugin/g

Reply:
repetitive at this level if we do it every time